swagger/v3/component_schemas/ECFParticipantTransfers.yml

```diff
@@ -0,0 +1,39 @@
+description: "The details of an ECF Participant enrolment"
+type: object
+required:
+  - training_record_id
+  - transfer_type
+  - status
+  - created_at
+properties:
+  training_record_id:
+    description: "The unique identifier of the participant training record"
```

Comment:
Do we want to add a line. "Should the DfE dedupe a participant, this value will not change."